apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx

```diff
@@ -1,15 +1,29 @@
-import React, { FunctionComponent } from 'react';
-import { Outlet } from 'react-router-dom';
+import React, { FunctionComponent, useEffect } from 'react';
+import { NavLink, Outlet, useLocation, useParams, useNavigate } from 'react-router-dom';
 import { t } from 'i18next';
 import { useAuthenticatedUserQuery } from '../../../../domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery';
 import { UserAvatar } from '../UserAvatar/UserAvatar';
+import { Tabs } from '@/common/components/organisms/Tabs/Tabs';
+import { TabsList } from '@/common/components/organisms/Tabs/Tabs.List';
+import { TabsTrigger } from '@/common/components/organisms/Tabs/Tabs.Trigger';
+import { TabsContent } from '@/common/components/organisms/Tabs/Tabs.Content';
 
 export const Home: FunctionComponent = () => {
   const { data: session } = useAuthenticatedUserQuery();
   const { firstName, fullName, avatarUrl } = session?.user || {};
+  const { locale } = useParams();
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+  const value = pathname.includes('workflows') ? 'workflows' : 'statistics';
+
+  useEffect(() => {
+    if (pathname === `/${locale}`) {
+      navigate('/en/statistics');
+    }
+  }, [pathname]);
```

Comment:
Note ESLint warns some items are missing from the dependency array.
```suggestion
  }, [locale, pathname, navigate]);
```